src/local/experiment.py

```diff
@@ -55,9 +56,11 @@ def start_mlflow_run(self):
     def load_data(self):
         self.logger.info("Loading data")
         
-        self.y_train = load_phenotype(self.cfg.data.phenotype.train)
-        self.y_val = load_phenotype(self.cfg.data.phenotype.val)
-        self.y_test = load_phenotype(self.cfg.data.phenotype.test)
+        self.y_train = load_phenotype(self.cfg.data.phenotype.train) - 160
```

Comment:
please don't hardcode 160; you could have a dictionary such as
start_val_pheno_dict = {'height': 160, 'asthma': ...}